app/views/__tests__/Import.spec.js

```diff
@@ -5,9 +5,13 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import Import from '../Import';
 
+const navigationMock = {
+  goBack: jest.fn()
+};
+
 it('renders correctly', () => {
   const tree = renderer
-    .create(<Import />)
+    .create(<Import navigation={navigationMock} />)
```

Comment:
A simple test of clicking  the right button opening a picker would be helpful, thanks.